osu.Game/Rulesets/Edit/DistancedHitObjectComposer.cs

```diff
@@ -75,36 +76,45 @@ protected override void LoadComplete()
             }
         }
 
-        protected override bool OnKeyDown(KeyDownEvent e)
+        public bool OnPressed(KeyBindingPressEvent<GlobalAction> e)
         {
-            if (!DistanceSpacingMultiplier.Disabled && e.ControlPressed && e.AltPressed && !e.Repeat)
+            switch (e.Action)
             {
-                RightSideToolboxContainer.Expanded.Value = true;
-                distanceSpacingScrollActive = true;
-                return true;
+                case GlobalAction.EditorIncreaseDistanceSpacing:
+                case GlobalAction.EditorDecreaseDistanceSpacing:
+                    return adjustDistanceSpacing(e.Action, 0.1f);
             }
 
-            return base.OnKeyDown(e);
+            return false;
         }
 
-        protected override void OnKeyUp(KeyUpEvent e)
+        public void OnReleased(KeyBindingReleaseEvent<GlobalAction> e)
         {
-            if (!DistanceSpacingMultiplier.Disabled && distanceSpacingScrollActive && (!e.AltPressed || !e.ControlPressed))
-            {
-                RightSideToolboxContainer.Expanded.Value = false;
```

Comment:
So the latest changes have sacrificed the old UX that would expand the toolbox when the distance spacing was being changed, but I think there's an avenue here to preserve it still. You could do something like expanding on every change, and adding a delayed scheduled delegate to contract the toolbox again in X ms. Subsequent changes within the X ms timeframe would cancel the delegate.

I'm bringing this up because now it becomes a bit less clear what is changing if you don't have the distance spacing grid enabled and don't know what to look for.

Reply:
Forgot to mention that, but I haven't bothered bringing this back since it'll improve with https://github.com/ppy/osu/pull/18071 (in which it would look weird to expand the slider). Can bring it back if you find that to be questionable UX, or for the sake of not intertwining things together.

Reply:
Oh I forgot a toast is coming. Maybe fine then.